util/logtools/make_timeline.ts

```diff
@@ -63,54 +62,7 @@ class ExtendedArgsRequired extends Namespace implements TimelineArgs {
 
 type ExtendedArgs = Partial<ExtendedArgsRequired>;
 
-// Some NPCs can be picked up by our entry processor.
-// We list them out explicitly here so we can ignore them at will.
-const ignoredCombatants = PetData['en'].concat([
-  '',
-  'Alisaie',
-  'Alisaie\'s Avatar',
-  'Alphinaud',
-  'Alphinaud\'s Avatar',
-  'Arenvald',
-  'Carbuncle',
-  'Carvallain',
-  'Crystal Exarch',
-  'Doman Liberator',
-  'Doman Shaman',
-  'Earthly Star',
-  'Emerald Carbuncle',
-  'Emerald Garuda',
-  'Estinien',
-  'Estinien\'s Avatar',
-  'G\'raha Tia',
-  'G\'raha Tia\'s Avatar',
-  'Gosetsu',
-  'Hien',
-  'Liturgic Bell',
-  'Lyse',
-  'Mikoto',
-  'Minfilia',
-  'Mol Youth',
-  'Moonstone Carbuncle',
-  'Obsidian Carbuncle',
-  'Raubahn',
-  'Resistance Fighter',
-  'Resistance Pikedancer',
-  'Ruby Carbuncle',
-  'Ruby Ifrit',
-  'Ryne',
-  'Thancred',
-  'Thancred\'s Avatar',
-  'Topaz Carbuncle',
-  'Topaz Titan',
-  'Urianger',
-  'Urianger\'s Avatar',
-  'Varshahn',
-  'Y\'shtola',
-  'Y\'shtola\'s Avatar',
-  'Yugiri',
-  'Zero',
-]);
+const ignoredCombatants = new EncounterCollector().ignoredCombatants;
```

Comment:
I was half envisioning making the  list of combatants a direct export from `encounter_tools`, but I think you're right, just adding the ignored combatants directly as collector properties makes more sense.

Reply:
> I was half envisioning making the list of combatants a direct export from `encounter_tools`, but I think you're right, just adding the ignored combatants directly as collector properties makes more sense.

I might end up doing it that way shortly.  It made sense originally as a collector property, but I'm toying around with adding a version of quisquous's "find interesting stuff" regex as an option in the splitter script/webtool, and I'm now thinking it might be handy to have it directly exported instead.